client/app/telecom/pack/xdsl/modem/service/modal/pack-xdsl-modem-service-modal.html

```diff
@@ -0,0 +1,11 @@
+<oui-modal data-heading="{{ 'xdsl_modem_service_modal_title' | translate }}"
+           data-primary-action="$ctrl.close()"
+           data-primary-label="{{:: 'common_confirm' | translate }}"
+           data-secondary-label="{{:: 'cancel' | translate }}"
+           data-secondary-action="$ctrl.cancel()"
+           data-on-dismiss="$ctrl.cancel()"
+           data-type="warning">
+    <oui-message type="warning" dismissible="false">
```

Comment:
Looks weird to have a `oui-message` in a `oui-modal` for just a message 
Also, there will be two warning icons

Reply:
You're right. I missed that.